Dockerfile

```diff
@@ -1,10 +1,8 @@
-FROM node:16-alpine
-LABEL "repository"="https://github.com/anothrNick/github-tag-action"
-LABEL "homepage"="https://github.com/anothrNick/github-tag-action"
-LABEL "maintainer"="Nick Sjostrom"
-
-RUN apk --no-cache add bash git curl jq && npm install -g semver
+# hadolint ignore=DL3007
+FROM ghcr.io/anothrnick/github-tag-action:latest
```

Comment:
How will this play out with versioning of the action? When a user just declares the action like this
```yaml
- name: Bump version and push tag
  uses: anothrNick/github-tag-action@1.55.0 # Don't use @master unless you're happy to test the latest version
  env:
    GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
    WITH_V: true
```

Will GHA build version `1.55.0` of the action? Or will it be `ghcr.io/anothrnick/github-tag-action:latest`?

Reply:
If we don't retro build all past tags as images they will be built on demand by the local workflow

Reply:
And moving forward the tagged images will be available?